packages/paste-website/src/pages/tokens/theme-package.mdx

```diff
@@ -0,0 +1,146 @@
+---
+title: Theme Package
+package: '@twilio-paste/theme'
+description: The Paste Theme Package allows easy and simple access to the design tokens via a theme provider to React components
+---
+
+import {graphql} from 'gatsby';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/theme/CHANGELOG.md';
+import {ThemeObjectDisplay} from '../../components/ThemeObjectDisplay.tsx';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteThemePackage(filter: {name: {eq: "@twilio-paste/theme"}}) {
+      edges {
+        node {
+          name
+          description
+          version
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Theme package"
+  categoryRoute={SidebarCategoryRoutes.TOKENS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/components/button"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/overview-theme--provider"
+  data={props.data.allPasteThemePackage.edges}
+/>
+
+***
+
+<content>
+
+## Theme
+
+Paste components are built using [Emotion](https://emotion.sh/) and [Styled System](https://styled-system.com) and providing a Theme object containing our design tokens. This enables our components to be themed, and also provide styling properties to our some of our design primtive components like `Box` and `Text`. The theme object is grouped by categories and for each category, each key represents a camelCase formatted design token name from the [tokens list](/tokens/).
+
+For examle the `$color-background-success` token would be available on the `backgrounds` key as `colorBackgroundSuccess`.
```

Comment:
This sentence is confusing.  Where is `backgrounds` used in the following example?  Isn't it the `backgroundColors` key?

Reply:
Yup